client/lib/performance-tracking/components/performance-tracker-install.jsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/no-danger */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+
+const performanceTrackerInstall = ( { nonce } ) => {
+	const serviceId = config( 'speedcurve_id' );
+	if ( ! config.isEnabled( 'rum-tracking/speedcurve' ) || ! serviceId ) return null;
+
+	return (
+		<>
+			<script
```

Comment:
Is it safe to return a script tag in a React component render? Is it guaranteed that it will never unexpectedly re-render and cause a new execution?

My initial inclination would be to avoid React altogether for this, and do a simple `document.createElement( 'script' )` instead, once.

Reply:
In this specific case I think it is safe because this component is only used by Node for the initial render. But I can see how that could be problematic.

Would it be ok to move this directly to `client/document/index.jsx`?

I don't think we can use `document.createElement()` because this will have to run in Node in the initial page rendering, so it should be something that works with `ReactDomServer.renderToString()`

Reply:
I wonder if there is a way to limit a component to only be rendered by ReactDomServer and not ReactDom?